docs/content/deployment/guides/kubernetes/customizing-your-deployment.mdx

```diff
@@ -130,7 +130,27 @@ Finally, the `dagster-user-deployments` subchart can now be managed in its own r
 
 ### Kubernetes Job and Pod TTL management
 
-If you use a Kubernetes distribution that supports the [TTL Controller](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/#ttl-controller), then `Completed` and `Failed` [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/) (and their associated [Pods](https://kubernetes.io/docs/concepts/workloads/pods/)) will be deleted after 1 day.
+If you use a Kubernetes distribution that supports the [TTL Controller](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/#ttl-controller), then `Completed` and `Failed` [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/) (and their associated [Pods](https://kubernetes.io/docs/concepts/workloads/pods/)) will be deleted after 1 day. The TTL value can be modified in your job tags:
+
+```python file=/deploying/kubernetes/ttl_config_job.py
+from dagster import job, op
+
+@op
+def my_op():
+    print("foo")
+
+@job(
+  tags = {
+    'dagster-k8s/config': {
+      'job_spec_config': {
+        'ttl_seconds_after_finished': 7200
+      }
+    }
+  }
+)
+def my_job():
+  my_op()
+```
```

Comment:
@johannkm to avoid syntax errors I needed to add quite a bit more boilerplate to the snipit.

Is there someway I can keep this tested with out the extra import/op or do you think thats worth keeping?

Reply:
You can add markers to the code so you can select only the relevant lines